[Australia] - Publish conference recap and links (#2071)
writethedocs/www: docs/conf/australia/2023/news/thanks-recap.rst

To ensure that our CoC was visible and accessible, we took a number of specific steps:

- Every participant agreed to the Code of Conduct in advance during registration.
- The CoC, or a summary, was included on the conference website and Hopin Reception
- The CoC was repeatedly mentioned in our conference introduction and announcements.
- We specifically stressed that the CoC applies to all conference spaces including the social event.
- We encouraged all attendees to report any incident, even if they were not sure whether it was a violation.

None of our organizers were approached about any CoC concerns at this event.
We know that this is, in part, a reflection of the thoughtful and inclusive community that we're so lucky to have.

That being said, we're also very conscious of the fact that just because nothing was reported doesn't mean nothing happened.
If there was any personal conduct or other aspect of the conference experience that did not meet the expectations of behaviour set out in the CoC, please email us at `conduct@writethedocs.org <mailto:conduct@writethedocs.org>`_.

Ready for more?
===============

> We are already in 2024 folks! Our `Portland conference <https://www.writethedocs.org/conf/portland/2024/>`__ is scheduled to happen 14-16 May and CFPs are now open.

Reviewer: I'd say "the CFP" or "the call for proposals"

You can `submit a talk for the Portland conference <https://www.writethedocs.org/conf/portland/2024/cfp/>`__ until **midnight Pacific time 15 January 2024**.
This conference will take place in-person in Portland.

If you cannot make the Portland conference, we will also announce dates for the online Atlantic conference soon.

> Around mid 2024, we will announce dates and more details for the Australian conference.

Reviewer: ```suggestion
We will announce dates and more details for the next Australian conference in the middle of 2024.
```

If you haven’t joined, now is the time! 
Check out the `Write the Docs Slack`_.

.. _Write the Docs Slack: http://www.writethedocs.org/slack/

Subscribe to our Newsletter
===========================

Our `monthly newsletter`_ is the other place to keep track of everything that is happening with the community. Subscribe to make sure you hear
about all the conferences, meetups, job postings, and summaries of relevant conversations in our Slack.

.. _monthly newsletter: http://writethedocs.org/newsletter/

Join a meetup
=============

We hope you continue to `build the meetup community`_ and keep in touch with other documentarians near your hometown. We have a number of
meetups in cities across the country, and if you don’t see your city listed consider `starting`_ a meetup!

.. _build the meetup community: http://www.writethedocs.org/meetups/
.. _starting: http://www.writethedocs.org/organizer-guide/meetups/starting/

Thanks again
============

We're so grateful to our sponsors, speakers, volunteers, and attendees for making this conference possible.

> Whether you were able to come out this time or not, we hope to see you again next year – or even sooner at one of our other events!

Reviewer: Were there sponsors? We should list them in this post?
